### PR TITLE
Use #loadAccounts to for deal, etch, and getNonce cheat codes

### PR DESCRIPTION
**Pull request by anvacaru**
Fixes: https://github.com/runtimeverification/foundry-tests/issues/27

### CODE REVIEW

**Review by tothtamas28 — APPROVED** [review]
LGTM. Please wait for approval from @ehildenb before merge.

**Review by ehildenb — APPROVED** [review]

**Pull request merged**
